pkg/services/factory.go

```diff
@@ -104,22 +105,14 @@ func (f *defaultFactory) GetKubeService() (kube.Kube, error) {
 }
 
 func (f *defaultFactory) GetGitClients(ctx context.Context, gpClient gitproviders.Client, params GitConfigParams) (git.Git, gitproviders.GitProvider, error) {
-	isExternalConfig := models.IsExternalConfigRepo(params.ConfigRepo)
```

Comment:
:question: Do we use this factory on the CLI?

Reply:
Yeah we do use this on the cli.